app/models/user.rb

```diff
@@ -18,6 +22,25 @@ def digest string
              end
       BCrypt::Password.create string, cost: cost
     end
+
+    def new_token
+      SecureRandom.urlsafe_base64
+    end
+  end
+
+  def remember
+    self.remember_token = User.new_token
+    update_attribute :remember_digest, User.digest(remember_token)
+  end
+
+  def authenticated? remember_token
+    return false if remember_digest.blank?
+
+    BCrypt::Password.new(remember_digest).is_password? remember_token
+  end
+
+  def forget
+    update_attribute :remember_digest, nil
```

Comment:
```suggestion
    update_column :remember_digest, nil
```

giair thích `update_attribute` và `update_column` khác nhau thế nào cu

Reply:
[update_attribute_&_update_column.odt](https://github.com/haupt-2855/sample_app/files/7768291/update_attribute_._update_column.odt)